### ISSUE TITLE
ERRORs on start-up Spigot 1.15

### ISSUE THREAD

**Issue by SlimeDog** [Shopkeepers/Shopkeepers#621]
* **Shopkeepers version:** `2.8.1`
* **Spigot version:** `1.15`

On server start-up:
```
[11:14:27] [Server thread/INFO]: [Shopkeepers] Loading Shopkeepers v2.8.1
[11:14:27] [Server thread/ERROR]: [Shopkeepers] Potentially incompatible server version: v1_15_R1
[11:14:27] [Server thread/ERROR]: [Shopkeepers] Shopkeepers is trying to run in 'compatibility mode'.
[11:14:27] [Server thread/INFO]: [Shopkeepers] Check for updates at: https://dev.bukkit.org/projects/shopkeepers/
[11:14:27] [Server thread/INFO]: [Shopkeepers] Loading config.
[11:14:27] [Server thread/INFO]: [Shopkeepers] Registering WorldGuard flag 'allow-shop'.
[11:15:27] [Server thread/INFO]: [Shopkeepers] Enabling Shopkeepers v2.8.1
[11:15:27] [Server thread/INFO]: [Shopkeepers] Config already loaded.
[11:15:27] [Server thread/INFO]: [Shopkeepers] Loading data of 1 shopkeepers..
[11:15:32] [Server thread/INFO]: +> [Depenizen] Depenizen loaded bridge for 'Shopkeepers'! 
```
These should be WARN, not ERROR.

**Comment by blablubbabc** [comment]
Will change it in the next version.

**Comment by blablubbabc** [comment]
Fixed with v2.9.0

**Comment by SlimeDog** [comment]
Looking forward to it. :)
